runtime/lib/DataAPI/mongoDataAPI.ts

```diff
@@ -61,6 +72,10 @@ class MongoDataAPI<
   };
 
   public getProject = async (projectID: string) => {
```

Comment:
looks like we should rename param
```suggestion
  public getProject = async (projectIDOrAuth: string) => {
```